public/electron.js

```diff
@@ -602,6 +607,18 @@ app.on('ready', async () => {
 });
 
 app.on('window-all-closed', () => {
+  // post sign out action
```

Comment:
Do you need to bootstrap the DB? Also, can you factor it out to its own handler?